pages/playback/controls/player/radiobutton-widget/basic-radio.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="div__radio-button-group-background">
+    <RadioButtonWidget :radio_buttons="button_names"></RadioButtonWidget>
+  </div>
+</template>
+
+<script>
+import { RadioButtonWidget } from "@/dist/mantarray.common";
+// import RadioButtonWidget from "@/components/playback/controls/player/RadioButtonWidget.vue";
+
+export default {
+  components: {
+    RadioButtonWidget,
+  },
+  data() {
+    return {
+      button_names: ["warm", "cool", "blue/red", "purple/green"],
```

Comment:
in general, we want our tests to check a wide range of possible parameters, especially if it doesn't require any additional tests to do so.  So for example it would be good to have a test or two have different number of button_names and different text for the button_names.  Whether it would be necessary to create a whole separate test just to look at that is debatable.  But in this case you already have 4 separate pages, so it is very easy to add in some diversity into the test cases

Reply:
sure.